common/modules/farm/events/src/events.rs

```diff
@@ -3,12 +3,12 @@
 elrond_wasm::imports!();
 elrond_wasm::derive_imports!();
 
-use common_structs::{DefaultFarmPaymentAttributesPair, FarmTokenAttributes};
-use contexts::{
-    claim_rewards_context::{ClaimRewardsContext, CompoundRewardsContext},
-    exit_farm_context::ExitFarmContext,
-    storage_cache::{FarmContracTraitBounds, StorageCache},
-};
+use common_structs::FarmTokenAttributes;
+// use contexts::{
```

Comment:
why is everything commented out ?

Reply:
Events were unused, and the compiler kept giving me warnings. Will integrate them again in a future PR.